EIPS/eip-4844.md

```diff
@@ -45,17 +45,18 @@ Compared to full data sharding, this EIP has a reduced cap on the number of thes
 | `BLOB_COMMITMENT_VERSION_KZG` | `Bytes1(0x01)` |
 | `POINT_EVALUATION_PRECOMPILE_ADDRESS` | `Bytes20(0x14)` |
 | `POINT_EVALUATION_PRECOMPILE_GAS` | `50000` |
-| `MAX_BLOBS_PER_BLOCK` | `16` |
-| `TARGET_BLOBS_PER_BLOCK` | `8` |
-| `MAX_BLOBS_PER_TX` | `2` |
-| `GASPRICE_UPDATE_FRACTION_PER_BLOB` | `64` |
+| `MAX_DATA_GAS_PER_BLOCK` | `2**21` |
+| `TARGET_DATA_GAS_PER_BLOCK` | `2**20` |
+| `MIN_DATA_GASPRICE` | `10**8` |
```

Comment:
This should be `0`, or if there is some mathematical reason it can't be (math often breaks at/near 0) then it should be whatever the smallest number that *doesn't* cause it to break is.